### PR TITLE
Task/add specific failed comment

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Update the error statement so if a user has a failed login more than 10 times, we can set up a cloudwatch alert for it.

Log while testing:

```
"{'result': 'error', 'message': {'password': ['Incorrect password for user_id ddd8b872-3012-46b4-85d7-24d7f8e8775a']}}" [in /workspace/app/errors.py:74]
2023-11-01T20:55:28 api api INFO None "{'result': 'error', 'message': {'password': ['Incorrect password for user_id ddd8b872-3012-46b4-85d7-24d7f8e8775a']}}" [in /workspace/app/errors.py:74]
2023-11-01T20:55:33 api api INFO None "{'result': 'error', 'message': {'password': ['Incorrect password for user_id ddd8b872-3012-46b4-85d7-24d7f8e8775a']}}" [in /workspace/app/errors.py:74]
2023-11-01T20:55:39 api api INFO None "{'result': 'error', 'message': {'password': ['Incorrect password for user_id ddd8b872-3012-46b4-85d7-24d7f8e8775a']}}" [in /workspace/app/errors.py:74]
2023-11-01T20:55:44 api api INFO None "{'result': 'error', 'message': {'password': ['Failed login: Incorrect password for user_id ddd8b872-3012-46b4-85d7-24d7f8e8775a failed_login 10 times
```

### CODE REVIEW

File: app/user/rest.py [hunk 1]
```diff
@@ -230,7 +230,14 @@ def verify_user_password(user_id):
         return jsonify({}), 204
     else:
         increment_failed_login_count(user_to_verify)
-        message = "Incorrect password"
+        # We want to get the user again to check the failed login count
+        user_to_verify = get_user_by_id(user_id=user_id)
```
Comment (andrewleith) [inline app/user/rest.py:234]: Why bother going back to the DB for the user again?

**Review by andrewleith — COMMENTED** [review]
Just a question about a possible db fetch that isn't needed.

File: app/user/rest.py (hunk 1 above)
Comment (andrewleith) [reply app/user/rest.py:234]: Right but I think the object is updated in place so I don't think we need to bother with the extra db call!

Comment (jzbahrai) [reply app/user/rest.py:234]: let me look at it again, thanks for pointing it out

**Review by andrewleith — APPROVED** [review]
LGTM!

**Pull request merged**